app/models/server_role.rb

```diff
@@ -5,32 +5,19 @@ class ServerRole < ActiveRecord::Base
   validates_presence_of     :name
   validates_uniqueness_of   :name
 
-  FIXTURE_DIR = File.join(Rails.root, "db/fixtures")
-
-  def self.seed_data
-    File.read(File.join(FIXTURE_DIR, "#{to_s.pluralize.underscore}.csv"))
-  end
-
   def self.seed
-    data = seed_data.split("\n")
-    cols = data.shift.split(",")
-
-    data.each do |a|
-      next if a =~ /^#.*$/ # skip commented lines
+    CSV.foreach(fixture_path, :headers => true, :skip_lines => /^#/).each do |csv_row|
+      action = csv_row.to_hash
+      action['license_required'] ||= ''
```

Comment:
Oh, and there is also [manageiq.pot](https://github.com/ManageIQ/manageiq/blob/master/config/locales/manageiq.pot#L15944-L15946)